samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -36,6 +36,9 @@ The sample supports only and exclusively the following development kits:
 
 .. _broadcast_configuration_tool_ui:
```

Comment:
Move this link tag above the **Shell commands list** title, not forgetting to leave one blank line before the title.
It is now failing the doc build. Link tags must always be just before the section titles.